src/main/java/com/exadel/discount/service/impl/VendorServiceImpl.java

```diff
@@ -32,6 +33,20 @@ public VendorDTO save(CreateVendorDTO vendorDTO) {
         return vendorMapper.getDTO(savedVendor);
     }
 
+    @Override
+    @Transactional
```

Comment:
Why here it is @Transactional and on category not?

Reply:
+